ffi_internals/src/struct_internals/struct_ffi.rs

```diff
@@ -156,6 +167,28 @@ impl<'a> From<StructFFI<'_>> for TokenStream {
         let free_fn_name = struct_ffi.free_fn_name();
         let init_fn_name = struct_ffi.init_fn_name();
         let clone_fn_name = struct_ffi.clone_fn_name();
+        let required_imports: Vec<Self> = struct_ffi.required_imports
+            .iter()
+            .map(|import| {
+                quote!(use #import;)
+            })
+            .collect();
```

Comment:
Ugh. Fixing this to do what `required_imports` says it will do causes some issues -- we're now using it to figure out what the consumer needs to import *and* what the ffi module needs to import, which is frequently not the same thing.

- [ ] Split these into two separate attributes -- `ffi_mod_imports` and `consumer_imports`.